### ISSUE TITLE
category views problem after magento 2.4.4 update

### ISSUE THREAD

**Issue by Danieleeffe1** [magepow/magento-2-categories#12]
magento installation 2.4.4
php 7.4
apache 2

dynamics:
updated with composer

problem:
if I enter any category, all the first and second level categories are displayed.

to see the problem reacars at the following address:
[problema image categories](https://m2.vetrinissime.it)

**Comment by clasteroid** [comment]
I encountered the same issues (tons of irrelevant categories show) on 2.4.3 and resolved with the following steps.

Open:
Magepow/Categories/Block/Categories.php

Editing line 141:
Original:
                            // ->addAttributeToFilter('parent_id', $categoryId)
                            
Revision:
                             ->addAttributeToFilter('parent_id', $categoryId)

I removed "//" which looks like a change made between the last couple versions of this module.
After the change only the relevant sub-categories shown as expected. 

Hope this helps or someone can explain why this line was commented out.
Thanks!
